ansible-devnet/genesis/validator-config.yaml

```diff
@@ -47,4 +47,15 @@ validators:
       ip: "46.224.135.177"
       quic: 9001
     metricsPort: 8081
-    count: 1
\ No newline at end of file
+    count: 1
+
+  - name: "lighthouse_0"
+    # node id a1b2c3d4e5f6a7b8c9d0e1f2a3b4c5d6e7f8a9b0c1d2e3f4a5b6c7d8e9f0a1b2
+    # peer id 16Uiu2HAm7TYVs6qvDKnrovd9m4vvRikc4HPXm1WyLumKSe5fHxBv
+    privkey: "4fd22cf461fbeae4947a3fdaef8d533fc7fd1ef1ce4cd98e993210c18234df3f"
+    # verify /ip4/127.0.0.1/udp/9004/quic-v1/p2p/16Uiu2HAm7TYVs6qvDKnrovd9m4vvRikc4HPXm1WyLumKSe5fHxBv
+    enrFields:
+      ip: "46.224.135.169"
```

Comment:
This IP is being used by zeam right now. Please backout the changes in this file. I'll handle it based on server availability.

Reply:
we already have one server that we can use @g11tech . But having another one does help. For ansible we need changes to other files as wel.

Reply:
Also, feel free to retain 46.224.135.169 for lighthouse. I have moved zeam to another server. @hopinheimer